src/tribol/utils/TestUtils.cpp

```diff
@@ -773,9 +773,9 @@ void TestMesh::allocateAndSetVelocities( int meshId, real valX, real valY, real
    // Check that mesh ids are not the same. The TestMesh class was built around 
    // testing the mortar method with Lagrange multiplier enforcement, which does not 
    // support auto contact.
-   SLIC_WARNING_IF( this->mortarMeshId == this->nonmortarMeshId, 
-                    "TestMesh::allocateAndSetVelocities(): please set unique " << 
-                    "mortarMeshId and nonmortarMeshId prior to calling this routine.");
+   SLIC_ERROR_IF( this->mortarMeshId == this->nonmortarMeshId, 
```

Comment:
Should the macros in this file be `SLIC_ERROR_ROOT_IF`?

Reply:
Debatable. This test utility class isn't designed to be MPI aware. using 'ROOT' probably is best for clarity, but the use of the macros is fine the way they are technically because this is meant for serial tests. This is outside the scope of this PR, but a good point.